### PR TITLE
CSS Modules: css colors

### PR DESCRIPTION
**Pull request by estelle**
DRAFT: not ready for review

depends on https://github.com/mdn/css-examples/pull/125 (merged)

note: `hue` and `css_color_adjustment` pages , showing as flawed links, are added in other PRs. The "image" making this PR fail is for https://github.com/mdn/content/pull/25350

### CODE REVIEW

File: files/en-us/web/css/css_colors/index.md [hunk 1]
```diff
@@ -2,42 +2,127 @@
...
+The **CSS color** module defines colors, color types, color blending, opacity, and how you can apply these colors and effects to HTML content.
 
-Not all CSS properties that take a {{cssxref("&lt;color&gt;")}} as a value are part of this module, but they do depend upon it.
+While this module only two CSS properties, {{cssxref("color")}}
+
+- {{cssxref("opacity")}}, over 20 CSS and SVG properties, CSS images, at-rules, and @media rules depend on the CSS color and opacity values defined in this module.
+
+### Colors in action
+
+To view a new color, along with the color value written in [red-green-blue](/en-US/docs/Web/CSS/color_value/rgb), [hexadecimal](/en-US/docs/Web/CSS/hex-color), [hue, saturation, and lightness](/en-US/docs/Web/CSS/color_value/hsl), and [hue, whiteness, and blackness](/en-US/docs/Web/CSS/color_value/hwb) CSS color formats, open the [color picker](/en-US/docs/Web/HTML/Element/input/color) and select a new color in the box below. Change the opacity of the color with the opacity [slider](/en-US/docs/Web/HTML/Element/input/range).
```
Comment (dipikabh) [inline files/en-us/web/css/css_colors/index.md:20]:
```suggestion
The box below contains a color syntax converter. When you choose a color and an opacity, it shows the corresponding values of the color in [red-green-blue](/en-US/docs/Web/CSS/color_value/rgb) (RGB), [hexadecimal](/en-US/docs/Web/CSS/hex-color) (HEX), [hue, saturation, and lightness](/en-US/docs/Web/CSS/color_value/hsl) (HSL), and [hue, whiteness, and blackness](/en-US/docs/Web/CSS/color_value/hwb) (HWB) CSS color formats.
To view a new color, click the color box next to "Select a color" and choose a color from the [color picker](/en-US/docs/Web/HTML/Element/input/color). Close the color picker and change the opacity of the color using the "Select an opacity" [slider](/en-US/docs/Web/HTML/Element/input/range). Notice the values for RGB, HEX, HSL, and HWB for the colors and opacity you choose.
```

**Review by dipikabh — CHANGES_REQUESTED** [review]

File: files/en-us/web/css/css_colors/index.md [hunk 2]
```diff
@@ -2,42 +2,127 @@
...
+
+{{EmbedGHLiveSample("css-examples/modules/colors.html", '100%', 450)}}
+
+To see the code for this color syntax converter, [view the source on Github](https://github.com/mdn/css-examples/blob/main/modules/colors.html).
```
[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/css/css_colors/index.md:24]: I changed it to reflect what is shown above the demo and how to change it below the demo. Is that better?

File: files/en-us/web/css/css_colors/index.md [hunk 3]
```diff
@@ -1,43 +1,128 @@
...
 
-**CSS Color** is a CSS module that deals with colors, color types, color blending, opacity, and how you can apply these colors and effects to HTML content.
+The **CSS color** module defines colors, color types, color blending, opacity, and how you can apply these colors and effects to HTML content.
 
-Not all CSS properties that take a {{cssxref("&lt;color&gt;")}} as a value are part of this module, but they do depend upon it.
+While this module has only two CSS properties, {{cssxref("color")}} and {{cssxref("opacity")}}, over 20 CSS and SVG properties, CSS images, at-rules, and @media rules depend on these two properties.
+
+### Colors in action
+
+This color syntax converter shows the values of the currently selected color in [red-green-blue](/en-US/docs/Web/CSS/color_value/rgb) (RGB), [hexadecimal](/en-US/docs/Web/CSS/hex-color) (HEX), [hue, saturation, and lightness](/en-US/docs/Web/CSS/color_value/hsl) (HSL), and [hue, whiteness, and blackness](/en-US/docs/Web/CSS/color_value/hwb) (HWB) CSS color formats. The RGB, HEX, HSL, and HWB color values, which written differently, all represent the same colors value.
```
Comment (estelle) [inline files/en-us/web/css/css_colors/index.md:18]:
```suggestion
This color syntax converter shows the values of the currently selected color in [red-green-blue](/en-US/docs/Web/CSS/color_value/rgb) (RGB), [hexadecimal](/en-US/docs/Web/CSS/hex-color) (HEX), [hue, saturation, and lightness](/en-US/docs/Web/CSS/color_value/hsl) (HSL), and [hue, whiteness, and blackness](/en-US/docs/Web/CSS/color_value/hwb) (HWB) CSS color formats. The RGB, HEX, HSL, and HWB color values, while written differently, all represent the same color value.
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Suggested a few edits

**Review by dipikabh — APPROVED** [review]
Thanks 👍

**Pull request merged**